### ISSUE TITLE
Regarding categorical evaluation

### ISSUE THREAD

**Issue by soham-joshi** [cshizhe/vil3dref#8]
@cshizhe  Great work by the team!

Hi @cshizhe ,
I had a query regarding evaluation. The inference of the model produces a val_outs.json file. This file has "scene_id" and its prediction like "object_ids" and corresponding logits. 
I wanted to run a baseline experiment (just like Table 6 in the paper Vil3DRef) and I wanted to compare the numbers. 
To do so, I would need the numbers in the "Easy", "Hard", "View-Dep", and "View-Independent" categories.

However, the inference log only outputs details like:
07/20/2023 10:02:29 - INFO - __main__ -   loss/og3d: 2.4535, loss/obj3d_clf: 3.2269, loss/obj3d_clf_pre: 2.1665, loss/txt_clf: 0.0000, loss/total: 8.7290, loss/cross_attn_0: 0.0023, loss/cross_attn_1: 0.0005, loss/cross_attn_2: 0.0006, loss/cross_attn_3: 0.0011, loss/self_attn_0: 0.2381, loss/self_attn_1: 0.0595, loss/self_attn_2: 0.0451, loss/self_attn_3: 0.0298, loss/hidden_state_0: 0.2778, loss/hidden_state_1: 0.0488, loss/hidden_state_2: 0.0482, loss/hidden_state_3: 0.0570, loss/hidden_state_4: 0.0733, acc/og3d: 0.7255, acc/og3d_class: 0.8635, acc/obj3d_clf: 0.6763, acc/obj3d_clf_pre: 0.5936, acc/txt_clf: 1.0000

Could you please help in evaluating the categorical scores for the categories like Easy, Hard, etc.?

Thanks in advance!
